src/__tests__/surveys.test.ts

```diff
@@ -400,4 +401,31 @@ describe('surveys', () => {
             })
         })
     })
+
+    describe('decide response', () => {
+        beforeEach(() => {
+            // clean the JSDOM to prevent interdependencies between tests
+            document.body.innerHTML = ''
+            document.head.innerHTML = ''
+        })
+
+        it('should not load when decide response says no', () => {
+            surveys.afterDecideResponse({ surveys: false } as DecideResponse)
+            // Make sure the script is not loaded
+            expect(checkScriptsForSrc('https://us-assets.i.posthog.com/static/surveys.js', true)).toBe(true)
+        })
+
+        it('should load when decide response says so', () => {
+            surveys.afterDecideResponse({ surveys: true } as DecideResponse)
+            // Make sure the script is loaded
+            expect(checkScriptsForSrc('https://us-assets.i.posthog.com/static/surveys.js')).toBe(true)
+        })
+
+        it('should not load when config says no', () => {
+            config.disable_surveys = true
+            surveys.afterDecideResponse({ surveys: true } as DecideResponse)
+            // Make sure the script is not loaded
+            expect(checkScriptsForSrc('https://us-assets.i.posthog.com/static/surveys.js', true)).toBe(true)
```

Comment:
fly-by... reading this for a second time I realised we pass `true` to mean "script is not present"

not this PR I know but that feels we should have `checkScriptIsLoaded` and checkScriptIsNotLoaded` or `checkScriptsForSrc({src: 'https://us-assets.i.posthog.com/static/surveys.js', shouldBePresent: false})`

Reply:
Yeaaaaaaaa

Reply:
Yeahhhh sorry, that's much nicer indeed 🙈

Reply:
That's what you get for letting me review it in the first place @neilkakkar :)

Reply:
(takes me a little while to get warmed up)

Reply:
Fixed this up whilst im here